src/info/author.rs

```diff
@@ -77,12 +77,22 @@ impl std::fmt::Display for AuthorsInfo {
         let mut authors_info = String::new();
 
         let pad = self.title().len() + 2;
+        let mut alignment_pad: usize = 0;
 
         for (i, author) in self.authors.iter().enumerate() {
             if i == 0 {
+                alignment_pad = author.to_string().find('%').unwrap();
                 write!(authors_info, "{author}")?;
             } else {
-                write!(authors_info, "\n{:<width$}{}", "", author, width = pad)?;
+                // Set padding based on top author
+                let author_pad = alignment_pad - author.to_string().find('%').unwrap();
+                write!(
+                    authors_info,
+                    "\n{:<width$}{}",
+                    "",
+                    author,
+                    width = pad + author_pad
```

Comment:
That's better @lukehsiao 👍 

We can try to make it cleaner and more readable, here is a suggestion:

-  Create two methods `top_contribution` in `AuthorsInfo` and `left_pad` in `Author`:
```rust
fn top_contribution(&self) -> usize {
    if let Some(top_contributor) = self.authors.get(0) {
        return top_contributor.contribution;
    }
    0
}
```
```rust
fn left_pad(&self, top_contribution: usize) -> usize {
    if top_contribution > 9 && self.contribution < 10 {
        return 1;
    }
    0
}
```

- in `fmt` of `AuthorsInfo` do `width = pad + author.left_pad(self.top_contribution())` when `i > 0`

Reply:
`usize` has `checked_ilog10` if you want to simplify this a bit.

Reply:
Done.